blog/2023-05-23-conda-23.5.0.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "May Releases for Conda and Conda-Build"
+slug: "2023-05-23-may-2023-releases"
+description: |
+    Conda 23.5.0 and conda-build 3.25.0 have been released! 🎉
+authors: [beeankha]
+tags: [announcement, conda]
+
+---
+
+May releases are here! 🎉 Both conda 23.5.0 and conda-build 3.25.0 are now available on both main and conda-forge:
+
+## [Changes in Conda](https://github.com/conda/conda/releases/tag/23.5.0)
+
+To update conda, run:
+```bash
+conda install -n base conda=23.5.0
+```
+
+### ✨ What's New? ✨
+
+The long-awaited [`conda doctor` subcommand plugin](https://github.com/conda-incubator/conda-dot-org/pull/133) has been implemented! The [related conda issue is over nine years old](https://github.com/conda/conda/issues/474) and was filed by the CEO of Anaconda himself, Peter Wang. The `conda doctor` command enables conda users to detect any packages with files missing (i.e., corrupt packages) in their conda environment.
+
```

Comment:
Bianca, think about adding
```<!-- truncate -->```
here.  That will cause only the parts above it to appear on the main blog page, and add a "see more" link at the bottom of this post.

Reply:
Hopefully I'm using `truncate` correctly!